bpf/lib/l3.h

```diff
@@ -57,6 +57,11 @@ static __always_inline int ipv4_l3(struct __ctx_buff *ctx, int l3_off,
 
 #ifndef SKIP_POLICY_MAP
 #ifdef ENABLE_IPV6
+/* Performs IPv6 L2/L3 handling and delivers the packet to the destination pod
```

Comment:
Maybe we shouldn't refer to "pod" but instead "endpoint"

Reply:
We already use both pod and endpoint in datapath comments.